nipype/interfaces/afni/tests/test_deconvolve.py

```diff
@@ -0,0 +1,10 @@
+"""Test afni deconvolve"""
```

Comment:
Please rename this file to "test_extra_Deconvolve".

This isn't yet a part of documentation, but we realized that lowercase tests like this block building the auto test on OSX. Using "extra" marks it as something to have in addition to the auto-test. And just for consistency's sake, let's use the capitals.